tests/mapper/test_node_mappers.py

```diff
@@ -365,11 +365,12 @@ def test_nodemapper_isolated_nodes():
     G = example_graph_3(feature_size=n_feat, n_nodes=6, n_isolates=1, n_edges=20)
 
     # Check connectedness
-    ccs = list(nx.connected_components(G))
+    Gnx = G._graph  # XXX Hack - Only works for NetworkXStellarGraph instances.
```

Comment:
This could be converted into an assertion, so that future changes have a clear prompt about what needs to change in this test, rather than having to decipher potentially weird error messages from networkx internals.

```suggestion
    assert isinstance(G, NetworkXStellarGraph) # XXX Hack - Only works for NetworkXStellarGraph instances
    Gnx = G._graph
```

But this is totally a minor change.

Reply:
Fair enough. I never liked this bit. Originally I put a method in StellarGraph to provide a NetworkX graph, but decided this was too much work to implement in general, so I took it out. The assertion works fine.

Reply:
Added assertion.